### DISCUSSION TITLE
[Privacy, Security] Does Artillery store or upload any data from local executions?

### DISCUSSION THREAD

**Discussion by kxngHunter** [artilleryio/artillery#2602] [Artillery]
I'm utilizing it within a project and simply seek some information to address any privacy or security concerns.

Thanks in advance.

**Comment by hassy** [comment]
Artillery does not upload any of your test scripts, custom code, payload files etc.

It collects some anonymized product analytics, but that can be turned off. There's a description of analytics collected here: https://www.artillery.io/docs/resources/telemetry

**Reply by kxngHunter** [reply]
Thank you @hassy
